.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: 'Create a report to help us improve the Agents SDK '
+title: ''
+labels: Triage
+assignees: ''
+
+---
+
+**Language/Version**
+What package version of the SDK are you using.
+- Language: 
+- Package version: 
+- SDK Use: Agents Hosting / Clients / Tools
+
+**if your using the Agent Hosting system, please complete the following information:**
+ - How are you Hosting this: [e.g. Azure, Desktop, Other]
+ - Are you deploying: [e.g. AppServices, Containers(type?)] 
```

Comment:
these two questions seems overlapping, when deployed the hosting is implicit. I dont think we care about Desktop.. this might be Cloud or OnPrem.